source/Player.cpp

```diff
@@ -59,9 +59,9 @@ namespace GTA
 	{
 		return Native::Function::Call<bool>(Native::Hash::IS_PLAYER_FREE_AIMING, this->Handle);
 	}
-	bool Player::IsOnMission::get()
+	bool Player::CanStartMission::get()
```

Comment:
Wait, so was this inverted the whole time? true in `CanStartMission` is not the same as true in `IsOnMission`